CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 - Generate Solidity documentation. (#111)
 - Migrate unit test files to Typescript & Hardhat:
     - IexecPoco2
+        - finalize (#117)
```

Comment:
Just found that we already had some changlog with finalize => #79, I don't know what to do with that ?

Reply:
Sure see [Update changelog](https://github.com/iExecBlockchainComputing/PoCo/pull/117/commits/18241d6fb9f4521aa7781b73036cfc63e2050afd)

Reply:
Perfect thanks